java/clients/src/test/java/sleeper/clients/admin/CompactionStatusReportScreenTest.java

```diff
@@ -0,0 +1,283 @@
+import static sleeper.clients.admin.testutils.ExpectedAdminConsoleValues.DISPLAY_MAIN_SCREEN;
+import static sleeper.clients.admin.testutils.ExpectedAdminConsoleValues.JOB_QUERY_ALL_OPTION;
+import static sleeper.clients.admin.testutils.ExpectedAdminConsoleValues.JOB_QUERY_DETAILED_OPTION;
+import static sleeper.clients.admin.testutils.ExpectedAdminConsoleValues.JOB_QUERY_RANGE_OPTION;
+import static sleeper.clients.admin.testutils.ExpectedAdminConsoleValues.JOB_QUERY_UNFINISHED_OPTION;
+import static sleeper.clients.admin.testutils.ExpectedAdminConsoleValues.MAIN_SCREEN;
+import static sleeper.clients.admin.testutils.ExpectedAdminConsoleValues.PROMPT_RETURN_TO_MAIN;
+import static sleeper.clients.admin.testutils.ExpectedAdminConsoleValues.TASK_QUERY_ALL_OPTION;
+import static sleeper.clients.admin.testutils.ExpectedAdminConsoleValues.TASK_QUERY_UNFINISHED_OPTION;
+import static sleeper.compaction.job.CompactionJobStatusTestData.jobCreated;
+import static sleeper.compaction.job.CompactionJobStatusTestData.startedCompactionRun;
+import static sleeper.configuration.properties.UserDefinedInstanceProperty.COMPACTION_STATUS_STORE_ENABLED;
+import static sleeper.configuration.properties.UserDefinedInstanceProperty.ID;
+import static sleeper.console.ConsoleOutput.CLEAR_CONSOLE;
+import static sleeper.console.TestConsoleInput.CONFIRM_PROMPT;
+import static sleeper.status.report.compaction.task.CompactionTaskStatusReportTestHelper.startedTask;
+
+class CompactionStatusReportScreenTest extends AdminClientMockStoreBase {
+    @Nested
+    @DisplayName("Compaction job status report")
+    class CompactionJobStatusReport {
+        private final CompactionJobTestDataHelper dataHelper = new CompactionJobTestDataHelper();
+        private final CompactionJobStatusStore compactionJobStatusStore = mock(CompactionJobStatusStore.class);
+
+        @Test
+        void shouldRunCompactionJobStatusReportWithQueryTypeAll() throws Exception {
+            // Given
+            createCompactionJobStatusStore();
+            when(compactionJobStatusStore.getAllJobs("test-table"))
+                    .thenReturn(exampleJobStatuses(dataHelper));
+
+            // When/Then
+            String output = runCompactionJobStatusReport()
+                    .enterPrompts(JOB_QUERY_ALL_OPTION, CONFIRM_PROMPT)
+                    .exitGetOutput();
+            assertThat(output)
+                    .startsWith(CLEAR_CONSOLE + MAIN_SCREEN + CLEAR_CONSOLE)
+                    .endsWith(PROMPT_RETURN_TO_MAIN + CLEAR_CONSOLE + MAIN_SCREEN)
+                    .contains("Compaction Job Status Report")
+                    .contains("" +
+                            "Total standard jobs: 1\n" +
+                            "Total standard jobs pending: 0\n" +
+                            "Total standard jobs in progress: 1\n" +
+                            "Total standard jobs finished: 0");
+
+            verifyWithNumberOfInvocations(4);
```

Comment:
I don't think this assertion is necessary. We're asserting which prompts will be made and that the output will end with a return to the main screen and displaying it once.

The fact that we've only specified an exit prompt once means that that must be the time that the client exited, or the main menu would be output multiple times, or it would throw an exception from TestConsoleInput when not enough prompts were provided.

I think that's the case for all the new tests, both for compaction and ingest. I think I'd rather avoid unnecessary use of Mockito.